src/nih_wayfinding/app/scripts/views/routing/directions/row-detail-directive.js

```diff
@@ -0,0 +1,55 @@
+(function () {
+    'use strict';
+
+     /* ngInject */
+    function RowDetail($compile, Directions) {
+        // TODO: Animate the row appearing
```

Comment:
@jfrankl I tried for a bit and failed :(

Pretty sure the ng-show/hide messes with things...insight welcome